Download.Mvc/Client/components/FolderView.tsx

```diff
@@ -38,12 +41,23 @@ export class FolderView extends React.Component<IProps, any> {
         };
     }
 
+    checkStatus(response) {
```

Comment:
@laholstege since you are using this in multiple places, make a little util module and you can use that from both places using an import.  Call it ApiUtils.js or something, and export the checkStatus function.  Then you can `import { checkStatus } from ApiUtils` and then you can dump the inline declaration from each of these places